src/main/java/hudson/plugins/parameterizedtrigger/BuildTriggerConfig.java

```diff
@@ -542,17 +609,28 @@ public FormValidation doCheckProjects(@AncestorInPath AbstractProject<?,?> proje
                 if (StringUtils.isNotBlank(projectName)) {
                 	Item item = Jenkins.getInstance().getItem(projectName,project,Item.class); // only works after version 1.410
                     if(item==null){
-                        return FormValidation.error(Messages.BuildTrigger_NoSuchProject(projectName,AbstractProject.findNearest(projectName).getName()));
+                        Item nearest = Items.findNearest(Job.class, projectName, Jenkins.getInstance());
+                        String alternative = nearest != null ? nearest.getRelativeNameFrom(project) : "?";
+                        return FormValidation.error(Messages.BuildTrigger_NoSuchProject(projectName, alternative));
                     }
-                    if(!(item instanceof AbstractProject)){
+                    if(!(item instanceof Job) || !(item instanceof ParameterizedJobMixIn.ParameterizedJob)) {
```

Comment:
Sure, for example a `Folder` is an `Item` but not a `Job`, and an [`ExternalJob`](https://github.com/jenkinsci/external-monitor-job-plugin/blob/c1725b597003f8ce1701e6e2fe83d6dd2f424671/src/main/java/hudson/model/ExternalJob.java#L45) is a `Job` but not a `ParameterizedJob`.